include/readdy/kernel/singlecpu/observables/SCPUObservableFactory.h

```diff
@@ -50,36 +48,44 @@ class SCPUKernel;
 namespace observables {
 
 class SCPUObservableFactory : public readdy::model::observables::ObservableFactory {
-
 public:
     explicit SCPUObservableFactory(readdy::kernel::scpu::SCPUKernel* kernel);
 
-    std::unique_ptr<readdy::model::observables::Energy> energy(Stride stride) const override;
+    [[nodiscard]] std::unique_ptr<readdy::model::observables::Energy>
+    energy(Stride stride, ObsCallBack<readdy::model::observables::Energy> callBack) const override;
```

Comment:
we should decide for one of `callback`, `call_back`, `callBack`, `CALLBACK`, `call_Back`...

Reply:
I would be in favor of `callback` actually

Reply:
true